osu.Game/Overlays/MusicController.cs

```diff
@@ -191,6 +191,7 @@ private void load(OsuGameBase game, OsuConfigManager config, BeatmapDatabase bea
 
             beatmapSource = game.Beatmap ?? new Bindable<WorkingBeatmap>();
             playList = beatmaps.GetAllWithChildren<BeatmapSetInfo>();
+            playList.RemoveAll(x => x.DeletePending);

```

Comment:
i mean sure... but this `playList` variable is never updated at runtime, so it's still going to lose consistency.